lib/dealership.rb

```diff
@@ -0,0 +1,72 @@
+class Dealership
+  attr_reader :name, :address, :inventory, :inventory_count, :dealership_value
+
+  def initialize(name, address)
+    @name = name
+    @address = address
+    @inventory = []
+    @inventory_count = 0
+    @dealership_value = 0
```

Comment:
I don't think you need `dealership_value` as an instance variable rather we could calculate this in a method when necessary. Also I don't think you are using it anywhere